### ISSUE TITLE
[Search] [Contract Spending] show the entity kind (company, NGO etc.)

### ISSUE THREAD

**Issue by akariv** [OpenBudget/budgetkey-app-search#25]
The data is in 'entity_kind'.

**Comment by danielbraun** [comment]
Waiting for @mushon for design decisions on this.

**Comment by mushon** [comment]
entity_kind is already designed and has been mocked up quite some time ago:
https://invis.io/MV7UO5T95#/162439872_Search_All
![image](https://user-images.githubusercontent.com/27123/29748237-c67b044e-8b1a-11e7-9c4c-dacdf4957a58.png)

**Comment by OriHoch** [comment]
I think this issue refers to entity kind as it's displayed on a procurement result

![image](https://user-images.githubusercontent.com/1198854/29750423-0d828b3c-8b48-11e7-8e41-942d72d71708.png)

**Comment by mushon** [comment]
Ok, I’m not working on it right now, so a bit hard to follow. Can we go over this face to face tomorrow?

**Comment by OriHoch** [comment]
I did a quick hack of the existing entity kind stamp from the entities items

![image](https://user-images.githubusercontent.com/1198854/29751259-b52bb88a-8b53-11e7-94fc-f28e88316c33.png)

my work in progress is available [here](https://gist.github.com/OriHoch/009bdad234fa5554db0afaf0aff609d6), it's not ready to merge yet, CSS requires some more work

let me know if it's in the right direction..

**Comment by OriHoch** [comment]
regarding meeting face to face - I'm afraid I don't usually go to the monday meetings..
